docfx/docs/migratingfromv1.md

```diff
@@ -236,8 +265,9 @@ If you previously had a lamda expression, you can simply add the extra arguments
 
 ```diff
 - btnLogin.Clicked += () => { /*do something*/ };
-+ btnLogin.Clicked += (s,e) => { /*do something*/ };
++ btnLogin.MouseClick += (s,e) => { /*do something*/ };
```

Comment:
The correct event is `Accept` in v2.

If a dev really wants to know that the button was acted upon by the user *with the mouse* (which would be a very bespoke case), the dev can subscribe to `MouseClick`.